test/test_ripper_tags.rb

```diff
@@ -95,7 +95,7 @@ module A::B
     assert_equal 'A::B::D', tags[2][:class]
   end
 
-  def test_extract_access
+  def test_extract_access_and_kind
```

Comment:
I think this test case is becoming too large. Maybe it would be a good idea to split different scenarios in different test cases? There are two things I'm interested in that don't seem to be covered by tests yet:

1. There's no test that verifies that after a `public/private/protected` keyword, **all** the following methods (not just the 1st one) inherit that access scope;

2. There is no test that verifies that the `public/private/projected def foo() end` syntax only changes the access scope for the method `foo` but not any subsequent methods (if this is indeed how Ruby works by design).